src/main/scala/com/cognite/sdk/scala/common/ReadWritableResource.scala

```diff
@@ -36,16 +36,14 @@ object DeleteByIds {
       _ => ()
     )
 
-  def deleteByIdsWithIgnoreUnknownIds[F[_]](
+  def deleteByCogniteIdsWithIgnoreUnknownIds[F[_]](
```

Comment:
```suggestion
  def deleteWithIgnoreUnknownIds[F[_]](
```

since there is no other overload now, maybe this shorter name would be fine. What do you think?

Reply:
done ;)